.github/workflows/ubuntu-ci.yml

```diff
@@ -39,21 +54,20 @@ concurrency:
 jobs:
   ubuntu-ci:
 
-    strategy:
-      fail-fast: false
-      matrix:
-        os-image:
-          - 'ubuntu-20.04'
-
     if: ${{ (
               !contains(github.event.pull_request.labels.*.name, 'skip-ci') &&
               !contains(github.event.pull_request.labels.*.name, 'conflicts')
             ) }}
     uses: ./.github/workflows/reusable-ubuntu-ci.yml
     with:
-      os-image: ${{ matrix.os-image }}
+      # It would be desirable to have a matrix of ubuntu OS for this job, but due to the issue opened in this ticket:
+      # https://github.com/orgs/community/discussions/128118 , it has been set as a single OS job.
+      os-image: 'ubuntu-22.04'
```

Comment:
Should we set `ubuntu 20.04`

Reply:
I overlooked it fixing the conflicts, nice catch!